com.vrcfury.vrcfury/Editor/VF/Inspector/VRCFuryHapticSocketEditor.cs

```diff
@@ -114,6 +114,15 @@ public class VRCFuryHapticSocketEditor : VRCFuryComponentEditor<VRCFuryHapticSoc
                 return activeBox;
             }, enableActiveAnimationProp));
 
+            var plugParams = VRCFuryEditorUtils.Section("Global Plug Parameters");

```

Comment:
Maybe outside the scope of this review, but I've always wanted to somehow integrate width into depth calculations. Perhaps by setting a "max width" box, that defines the width the animation is designed for, and we'd automatically scale the animation value by (actual width / defined max width)